### PR TITLE
7.2 | LPS-90869 + LPS-911980 Aggregations combined

### PR DESCRIPTION
**Pull request by lipusz**
https://issues.liferay.com/browse/LPS-90869

### CODE REVIEW

File: developer/frameworks/articles/160-search/xx-aggregations/03-statistical-aggregations.markdown [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+3. Get the desired metric, for example _cardinality_:
+
+```java
+        statsResponse.getCardinality();
+```
+
+**Example:** https://github.com/liferay/liferay-portal/blob/7.2.x/modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/stats/BaseStatisticsTestCase.java#L128
+
+<!-->
+### Using the Legacy `Stats` Object
```
Comment (rbohl) [inline developer/frameworks/articles/160-search/xx-aggregations/03-statistical-aggregations.markdown]: @lipusz If this is just a "if you prefer" thing, I'd rather leave it out. We want to show people the new (and better, hopefully) way of doing things. Do you agree?

**Comment by rbohl** [comment]
I think this is missing a few commits from my Cardinality branch :grin: 

https://github.com/rbohl/liferay-docs/commits/LPS-91190-cardinality-aggregation/draft.markdown

I already pull --rebased your branch into my aggregations branch, so I'll see what the best way to bring in the missing commits might be (maybe cherrypicking them?) or just apply the edits manually.

File: developer/frameworks/articles/160-search/xx-aggregations/03-statistical-aggregations.markdown (hunk 1 above)
Comment (lipusz) [reply developer/frameworks/articles/160-search/xx-aggregations/03-statistical-aggregations.markdown]: I'd keep it as it's not deprecated, means, it's fully supported.

**Comment by lipusz** [comment]
It's included. See this commit: https://github.com/rbohl/liferay-docs/pull/112/commits/ac13ce36b5234704cc1018d1bafeab7d6b26ba6f

@rbohl

**Comment by rbohl** [comment]
Merged locally, ready for review by developers outside the team.

Current work is here: https://github.com/rbohl/liferay-docs/tree/LPS-90869-new-aggregations/developer/frameworks/articles/160-search

**Pull request closed without merging**